aspnetcore/fundamentals/metapackage.md

```diff
@@ -60,3 +60,25 @@ Any dependencies of the preceding packages that otherwise aren't dependencies of
 
 * `StackExchange.Redis` as a dependency of `Microsoft.Extensions.Caching.Redis`
 * `Microsoft.ApplicationInsights` as a dependency of `Microsoft.AspNetCore.ApplicationInsights.HostingStartup`
+
+## Update ASP.NET Core 2.1
+
+We recommend you migrate to the `Microsoft.AspNetCore.App` metapackage. To Update `Microsoft.AspNetCore.App` to ASP.NET Core 2.1:
```

Comment:
Isn't this covered in metapackage-app.md? I'm assuming this section should be talking about updating Microsoft.AspNetCore.All?